testscripts/stor_add_nvme_disk.ps1

```diff
@@ -0,0 +1,191 @@
+###############################################################################
+##
+## Description:
+## Test disks works well when stor_add_nvme_disk
+##
+###############################################################################
+##
+## Revision:
+## v1.0.0 - ldu - 10/17/2019 - Build scripts.
+##
+## 
+###############################################################################
+
+<#
+.Synopsis
+    stor_add_nvme_disk
```

Comment:
stor_add_nvme_disk -> [virtual storage]Add a NVMe controller and disk to the Guest